src/System.Net.Security/src/System/Net/CertWrapper.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+
+namespace System.Net
+{
+    internal static class CertWrapper
```

Comment:
I don't really understand the purpose of this indirection. What is the design intent?